### ISSUE TITLE
word-count: test suite is missing which causes Travis CI build to fail

### ISSUE THREAD

**Issue by m-a-ge** [exercism/bash#77]
From Travis CI logs:
```
travis_time:end:03762840:start=1502581992494811503,finish=1502581992868551731,duration=373740228
[0K
[32;1mThe command "bin/fetch-configlet" exited with 0.[0m
travis_time:start:0f0d1e20
[0K$ bin/configlet lint .
-> The implementation for 'word-count' is missing a test suite.

travis_time:end:0f0d1e20:start=1502581992873047268,finish=1502581992880369493,duration=7322225
[0K
[31;1mThe command "bin/configlet lint ." exited with 1.[0m

Done. Your build exited with 1.
```

We should add `word_count_test.sh` test suite ASAP to fix this issue

**Comment by kotp** [comment]
Thank you @m-a-ge

**Comment by m-a-ge** [comment]
I am not that good with `bash`, so I need more time to finish #78

**Comment by kotp** [comment]
Closed via #78
